modules/aws_service.py

```diff
@@ -117,10 +117,14 @@ def get_apigateway_endpoint(config):
     return 'error', 1
 
 
-def upload_file_s3_bucket(file_name, results, test_file):
+def upload_file_s3_bucket(file_name, results, test_file, isArchive):
 
     s3_client = boto3.client('s3')
-    response = s3_client.upload_file(file_name, 'attack-range-attack-data', str(test_file['simulation_technique'] + '/attack_data.tar.gz'))
+    if isArchive:
+        response = s3_client.upload_file(file_name, 'attack-range-attack-data', str(test_file['simulation_technique'] + '/attack_data.tar.gz'))
+    else:
+        response = s3_client.upload_file(file_name, 'attack-range-attack-data', str(test_file['simulation_technique'] + '/attack_data.json'))
```

Comment:
same super minor: this string cast seems a bit brittle maybe store the variables for the directory path before the line?